docs/dashboard.asciidoc

```diff
@@ -3,163 +3,109 @@
 
-To reposition a visualization:
+[role="screenshot"]
+image:images/Dashboard_Resize_Menu.png[Example dashboard]
 
-. Hover over it to display the container controls.
-. Click and hold the *Move* button in the upper right corner of the container.
-. Drag the container to its new position. 
-. Release the *Move* button.
+[[moving-containers]]
+To move a visualization, click and hold the
+header of a visualization and drag to the new location.
 
-[float]
 [[resizing-containers]]
-==== Resizing Visualizations
-
-To resize a visualization:
-
-. Hover over it to display the container controls.
-. Click and hold the *Resize* button in the bottom right corner of the container.
-. Drag to change the dimensions of the container. 
-. Release the *Resize* button.
+To resize a visualization, click the resize control on the lower right and drag
+to the new dimensions.
 
-[float]
 [[removing-containers]]
-==== Removing Visualizations
+Additional commands for managing the
+container and its contents are in the gear menu in the upper right.
+
+NOTE: Deleting a visualization from a
+dashboard does _not_ delete the saved visualization.
 
-To remove a visualization from the dashboard:
 
-. Hover over it to display the container controls.
-. Click the *Delete* button in the upper right corner of the container.
-+
-NOTE: Removing a visualization from a dashboard does _not_ delete the
-saved visualization.
 
-[float]
 [[viewing-detailed-information]]
-=== Viewing Visualization Data
+== Viewing Visualization Data
 
-To display the raw data behind a visualization:
+You can display the raw data behind a visualization in a dashboard using the
+Expand/Collapse button.
 
-. Hover over it to display the container controls.
-. Click the *Expand* button in the lower left corner of the container.
-This displays a table that contains the underlying data. You can also view
-the raw Elasticsearch request and response in JSON and the request statistics.
-The request statistics show the query duration, request duration, total number
-of matching records, and the index (or index pattern) that was searched.
+. Open the dashboard.
+. Hover over the visualization to display the *Expand* button in the lower
+left.
+. Click *Expand* to display a table that contains the underlying data.
 +
-image:images/NYCTA-Table.jpg[]
+You can also view
+the raw Elasticsearch request, the response in JSON, and the request statistics.
+These statistics show the query duration, request duration, total number
+of matching records, and the index that was searched.
++
+[role="screenshot"]
+image:images/Dashboard_visualization_data.png[Example dashboard]
 
-To export the data behind the visualization as a comma-separated-values
-(CSV) file, click the *Raw* or *Formatted* link at the bottom of the data 
-Table. *Raw* exports the data as it is stored in Elasticsearch. *Formatted*
-exports the results of any applicable Kibana <<managing-fields,field
+To export the data as a comma separated values
```

Comment:
Hm. I thought it was pretty clear that by "raw data", she meant the data that was fed into the aggregation, not the original docs themselves, e.g. the aggregation data. Maybe, instead of "raw data", it should say something like:

"To export the visualization data" or "To export the aggregated data displayed in a visualization..." Dunnoes. Writing is hard.